messages/src/lib.rs

```diff
@@ -140,39 +138,93 @@ impl TryFrom<&Url> for url::Url {
     }
 }
 
-/// DAP protocol message representing a duration with a resolution of seconds.
+/// DAP protocol message representing a duration as a multiple of the task's time precision.
```

Comment:
This isn't quite right. If we have a time precision 5, then 0, 5, 10, 15, ... are multiples, but we'd represent the time as 0, 1, 2, 3.